decidim-core/app/views/decidim/participatory_processes/show.html.erb

```diff
@@ -73,6 +73,9 @@
   <div class="row">
     <div class="columns section view-side mediumlarge-4 mediumlarge-push-8 large-3 large-push-9">
       <%= render partial: "decidim/shared/share_modal" %>
+
+      <% embed_url = embedded_code_for(participatory_process_participatory_process_widget_path(current_participatory_process)) %>
```

Comment:
You should be using `blabla_url` instead of `_path`, otherwise you are returning an invalid HTML:

![](https://i.imgur.com/3t30Cx2.png)